emission/tests/modellingTests/modellingTestAssets.py

```diff
@@ -1,35 +1,145 @@
 import random
 from typing import Optional, Tuple, List, Dict
 from uuid import UUID
-import emission.analysis.modelling.trip_model.trip_model as eamtm
+import emission.analysis.modelling.trip_model.greedy_similarity_binning as eamtg
+import emission.tests.modellingTests.modellingTestAssets as etmm
 import emission.core.wrapper.confirmedtrip as ecwc
 
 import emission.core.wrapper.entry as ecwe
 import time 
 import math
 
+def setModelConfig(metric,threshold,cutoff,clustering_way,incrementalevaluation):
+    """
+    TODO: Write about each parameter to the function
+    pass in a test configuration to the binning algorithm.
+    
+    clustering_way : Part of the trip used for checking pairwise proximity.
+                        Can take one of the three values:
+                        
+                        1. 'origin' -> using origin of the trip to check if 2 points
+                                        lie within the mentioned similarity_threshold_meters
+                        2. 'destination' -> using destination of the trip to check if 2 points
+                                            lie within the mentioned similarity_threshold_meters
+                        3. 'origin-destination' -> both origin and destination of the trip to check 
+                                                if 2 points lie within the mentioned 
+                                                    similarity_threshold_meters
+    """        
+    model_config = {
+        "metric": metric,
+        "similarity_threshold_meters": threshold,  # meters,
+        "apply_cutoff": cutoff,
+        "clustering_way": clustering_way,  
+        "incremental_evaluation": incrementalevaluation
+    }
+
+    return eamtg.GreedySimilarityBinning(model_config)
+
+def generate_random_point():
+    """Generate a completetly random point valid WGS84 latitiude and longtidude"""
+    lat=random.uniform(-90,90)
+    lon=random.uniform(-180,180)
+    return [lat,lon]
```

Comment:
why do we want to generate a completely random point? We could still specify a centroid like we did before, right?

Reply:
From the next commit f5944ccef1140db347211c93b851b8864c0362ee , this will be used  when 

1.no ref. points are passed.
2. when we have to generate points that need to be outside the threshold.

 Currently, I do pass (0,0) and (1,1) as ref points.